### ISSUE TITLE
Why it does not converge when I train stanford_car with VGG_16? (Using caffe)

### ISSUE THREAD

**Issue by yangjingyi** [BVLC/caffe#5523]
I train the standford car dataset using VGG_16 net on caffe. I just finetune last layer. I even put learning rate as 0.0001 and gamma 0.00005. However, it dose not converge. The training loss have not decreased under 1.0 by over 2000 iteration. What may be the problem?

**Comment by williford** [comment]
Please ask your question on the [Caffe user mailing list](https://groups.google.com/forum/#!forum/caffe-users) and/or http://stackoverflow.com/.

> Please use the [caffe-users list](https://groups.google.com/forum/#!forum/caffe-users) for usage, installation, or modeling questions, or other requests for help.
> _Do not post such requests to Issues._ Doing so interferes with the development of Caffe.
